auth-example/auth.properties

```diff
@@ -0,0 +1,3 @@
+security.protocol=SASL_PLAINTEXT
+sasl.mechanism=PLAIN
+sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="my-user" password="my-password";
```

Comment:
This doesn't look valid? shouldn't it be one field per line and no semi colon?

https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html

Reply:
Well, actually it is valid event if it is not following properties file standard :)
https://docs.confluent.io/current/kafka/authentication_sasl.html#jaas-configurations

I just have chosen this way to have less files involved. Otherwise I will have to have another file with content from this property

Reply:
👍